lib/lunchbot/lunchbot_data/order.ex

```diff
@@ -23,4 +24,21 @@ defmodule Lunchbot.LunchbotData.Order do
   def get_orders_by_user(query, user_id) do
     from(q in query, where: q.user_id == ^user_id)
   end
+
+  def get_order_total_price(order_id) do
+    order =
+      Lunchbot.Repo.get_by(Lunchbot.LunchbotData.Order, id: order_id)
+      |> Repo.preload(order_items: [:item, order_item_options: [:option]])
+
+    get_total(order)
+  end
+
+  def get_total(%Lunchbot.LunchbotData.Order{} = order) do
+    total_items_price =
```

Comment:
In elixir, the last line of code is the return.  In this case the variable isn't needed.  You could write instead:
```
    def get_total(order) do
           Enum.map(order.order_items, &Lunchbot.LunchbotData.OrderItem.get_total/1)
           |> Enum.sum()
     end
```